packages/calypso-build/webpack.config.js

```diff
@@ -107,7 +110,7 @@ function getWebpackConfig(
 				global: 'window',
 			} ),
 			new webpack.IgnorePlugin( /^\.\/locale$/, /moment$/ ),
-			...SassConfig.plugins( { cssFilename, minify: ! isDevelopment } ),
+			...SassConfig.plugins( { cssChunkFilename, cssFilename, minify: ! isDevelopment } ),
```

Comment:
This `css*` pattern seems odd to me.

```suggestion
			...SassConfig.plugins( {
				chunkFilename: cssChunkFilename,
				filename: cssFilename,
				minify: ! isDevelopment,
			} ),
```

Reply:
I agree, it's clear to have `cssFilename` outside the shard but inside the shard it's the only "filename" so makes sense to drop `css*`.

Reply:
I agree, too. I didn't want to change the shard's interface with this PR since I think some consumers use it directly, so it constitutes a breaking API change.

Reply:
It's beta! 🤷‍♂️ 😅 😬